auth_oidc/models/auth_oauth_provider.py

```diff
@@ -50,6 +50,46 @@ class AuthOauthProvider(models.Model):
         help="Additional parameters for the auth link. "
         "For example: {'prompt':'select_account'}"
     )
+    oidc_discovery_url = fields.Char(string="OIDC config URL")
```

Comment:
```suggestion
    oidc_discovery_url = fields.Char(string="OIDC Discovery Document URL")
```

To use the standard terminology.